addons/im_livechat/tests/common.py

```diff
@@ -28,7 +27,11 @@ def setUpClass(cls):
             'name': 'Georges',
             'login': 'georges'
         }])
-
+        for operator in cls.operators:
+            cls.env['res.users.settings']._find_or_create_for_user(operator)
```

Comment:
Same remark, there should be no diff on this test, writing on the related should just work out of the box.